en/docs/modules/output/Lcd.md

```diff
@@ -19,8 +19,23 @@ display.Text = "Hello! How are  you today?"
 ```
 ![Text on an Lcd](../../../assets/docs/Lcd/Lcd.png)
 
+#### Examples
```

Comment:
Considering there's only one, and this isn't a top-level category, maybe change this to "Formatting" or something? Might make it easier to find for people quickly skimming.
```suggestion
#### Formatting lines automatically
```